File: BaoSheng/BaoShengMES-Static/WebContent/app/bas/propConfig/model/PropConfig.js
//PropConfig
Ext.define('bsmes.model.PropConfig', {
	extend : 'Ext.data.Model',
	fields : [ 
	    'id', 
	    'keyK',
	    'valueV',
	    'description',
	    {
	    	name : 'status',
			type : 'string',
			renderer : function(value) {
				if (value == 'true') {
					return Oit.msg.bas.prop.normal;
				} else if (value == 'false') {
					return Oit.msg.bas.prop.freeze;
				} else {
					return "";
				}
			}
	    },
	    'createUserCode',
	    {
	    	name : 'createTime',
	    	type : 'date'
	    },
	    'modifyUserCode',
	    {
	    	name : 'modifyTime',
	    	type : 'date'
	    }
	 ]
});

File: BaoSheng/BaoShengMES-Core/src/main/java/cc/oit/bsmes/bas/model/PropConfig.java
package cc.oit.bsmes.bas.model;

import cc.oit.bsmes.common.model.Base;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.ToString;

import javax.persistence.Table;

@Data
@EqualsAndHashCode(callSuper = false)
@ToString(callSuper = true)
@Table(name = "T_BAS_PROPERTIES")
public class PropConfig extends Base{

	/**
	 * 
	 */
	private static final long serialVersionUID = 4138687810799090249L;

	private String keyK;   //KEY_K  键
	private String valueV;  //VALUE_V  值
	private String description;   //DESCRIPTION  描述
	private Boolean status;   //STATUS   状态 
}
